app/views/posts/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: post,class: "form-horizontal",style:'margin:auto',local: true) do |form| %>
+<%= form_with(model: post,class: "form-horizontal",id:"fieldContainerForm",style:'margin:auto',local: true) do |form| %>
```

Comment:
I think for for html class or id name. Name can have following format e.g
`field-container-form` or `field_container_form`
As above is convention of naming JavaScript variables

Reply:
OK,I will keep in mind.

Reply:
I will follow this naming conventions in my next tasks.